app/scripts/templates/settings/account_recovery/account_recovery.mustache

```diff
@@ -0,0 +1,54 @@
+<div id="account-recovery-section" class="settings-unit {{#isPanelOpen}}open{{/isPanelOpen}} account-recovery-panel">
+    <div class="settings-unit-stub">
+        <header class="settings-unit-summary">
+            <h2 class="settings-unit-title">{{#t}}Account recovery{{/t}}</h2>
+        </header>
+
+        {{^hasRecoveryKey}}
+            <button class="settings-button primary-button settings-unit-toggle" data-href="settings/account_recovery">
+                {{#t}}Add…{{/t}}
+            </button>
+        {{/hasRecoveryKey}}
+
+        {{#hasRecoveryKey}}
+            <button class="settings-button secondary-button settings-unit-toggle" data-href="settings/account_recovery">
+                {{#t}}Change…{{/t}}
+            </button>
+        {{/hasRecoveryKey}}
+    </div>
+
+
+    <div class="settings-unit-details">
+        <p>
+            {{#unsafeTranslate}}Your Firefox Account password is used to secure all of the information you’ve saved to Firefox. If you need to reset your password, you’ll need a recovery key to regain access to this information. <a>Learn more</a>{{/unsafeTranslate}}
```

Comment:
SUMO article is being drafted [here](https://docs.google.com/document/d/1WgEE6CzTrYj6MEr_4xRRnH90jGqvggGxo35ROwfLZbg/edit?usp=sharing), once ready this link can be updated.